src/lib/types.d.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2024, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+export type Inputs = {
```

Comment:
Ideally I wanted to use the "Inputs" declaration from sfdx-scanner's types.d.ts file. But I can't access that file's declarations from vs code. That's also the reason for the compilation errors I ran into locally and in the build.

Reply:
Hmmm... that's odd. Maybe its because it is named types.d.ts instead of types.ts (not sure why it was named that way). But theoretically we should be able to export he "Inputs"... but this is fine to.

Reply:
This is now taken care of.